data/contract_docs/contract.md

```diff
@@ -50,6 +50,12 @@ There are two named cases for when a response is lacking data and also doesn't h
 
 We retrieve one or more data items from a datasource but an item has no values, that item is said to be an "**_Empty value_**". In this case, the required dataframe fields should still be present (but the fields themselves each have no values).
 
+## Error Responses
+
+Errors are returned from outside the dataframes using the `Error` property of a [DataResponse](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go/backend#DataResponse).
```

Comment:
should this describe the type of information to include in an error message? for example, should a developer return the raw error message from an API call, or should the messages always be constructed by the data source code to avoid exposing sensitive info?

Reply:
Also looks like https://github.com/grafana/grafana-plugin-sdk-go/pull/544 will be merged soon

Reply:
@toddtreece Good questions, but don't think I want to get into that here (at least not yet). Rather see what comes out of error plane and other error handling across Grafana.

@ryantxu Yup, I have that in mind, had mentioned in the PR notes.